<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Laminate Sketch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>158</width>
        <height>482</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5,5">
      <item>
       <widget class="QPushButton" name="btn_open_file">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Open File</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save_file">
        <property name="text">
         <string>Save File</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_save_settings">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled/>
         </palette>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Save File Settings:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,2">
         <item>
          <widget class="QComboBox" name="cb_format_selection">
           <item>
            <property name="text">
             <string>AC1027 (ACAD 2013)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AC1024 (ACAD 2010)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AC1021 (ACAD 2007)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AC1018 (ACAD 2004)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AC1015 (ACAD 2000)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_is_binary">
           <property name="text">
            <string>Binary</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,100">
      <item>
       <widget class="QLabel" name="lbl_message">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled/>
         </palette>
        </property>
        <property name="text">
         <string>Message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_message_text">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled/>
         </palette>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
